android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -444,13 +444,13 @@ public void addBreadcrumb(final ReadableMap breadcrumb) {
                     case "debug":
                         breadcrumbInstance.setLevel(SentryLevel.DEBUG);
                         break;
-                    case "error":
-                        breadcrumbInstance.setLevel(SentryLevel.ERROR);
-                        break;
                     case "info":
-                    default:
                         breadcrumbInstance.setLevel(SentryLevel.INFO);
                         break;
+                    case "error":
+                    default:
```

Comment:
H: `info` is default.

Reply:
Thanks, I've mixed up the breadcrumbs level and event level.